### PR TITLE
bump dash >=1.20.0

### PR DESCRIPTION
**Pull request by richarms**
Neglected to update requirements.txt to update version of dash (>=1.20.0) during https://skaafrica.atlassian.net/browse/SPR1-868

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,12 +8,11 @@ aioredis
 aiozk==0.14.0
 async-timeout
 click==7.0                 # via Flask
-dash==1.0.2
-dash-core-components==1.0.0
-dash-html-components==1.0.0
-dash-table==4.0.2
-dash-dangerously-set-inner-html==0.0.2
-dash-renderer==1.0.0       # via dash
```
Comment (bmerry) [inline requirements.txt:16]: Interesting that this vanishes - did it get absorbed into dash itself?

File: setup.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
         'aioredis',
         'aiozk',
         'async_timeout',
-        'dash',
+        'dash>=1.18.*',
```
Comment (bmerry) [inline setup.py:33]: Not sure that `*` makes logical sense with `>=`, but presumably pip is happy enough or this wouldn't have built). I would have said `dash>=1.18`.

**Review by bmerry — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (richarms) [reply requirements.txt:16]: yes, it's obsolete -- merged into the main dash repo

File: setup.py (hunk 2 above)
Comment (richarms) [reply setup.py:33]: noted

**Pull request merged**